src/services/user.service.js

```diff
@@ -0,0 +1,62 @@
+import models from '../models';
+
+const { Users } = models;
+
+/**
+ * @exports
+ * @class UserService
+ */
+class UserService {
+  /**
+ * create new user
+ * @static
+ * @param {object} newUser
+ * @memberof UserService
+ * @returns {object} data
+ */
+  static addUser(newUser) {
+    return Users.create(newUser);
+  }
+
+  /**
+ * find user by email
+ * @static
+ * @param {object} userEmail
+ * @memberof UserService
+ * @returns {object} data
+ */
+  static findByEmail(userEmail) {
+    return Users.findOne({
+      where: {
+        email: userEmail
+      }
+    });
```

Comment:
you should change the name of the findByEmail function here too, and also in es6 syntax, there is no need to call the value of the key if they are the same. in findByEmail by email as parameter then return findOne where email